addons/l10n_tw_chart/untitled.xml

```diff
@@ -0,0 +1,469 @@
+<record id="chart8020" model="account.account.template">
```

Comment:
missing tag `<?xml version="1.0" encoding="utf-8"?><odoo>` at the beginning of the file